reposerver/repository/repository.go

```diff
@@ -2079,26 +2080,25 @@ func loadFileIntoIfExists(path pathutil.ResolvedFilePath, destination *string) e
 	return nil
 }
 
-func findHelmValueFilesInPath(path string) ([]string, error) {
-	var result []string
+func walkHelmValueFilesInPath(root string, valuesFiles *[]string) filepath.WalkFunc {
```

Comment:
only nit is argument name should probably be `valueFiles` to match the other singular `value` usages in this file.  Otherwise great PR.

Reply:
fixed :)